src/jit/block.cpp

```diff
@@ -1357,3 +1357,143 @@ BasicBlock* Compiler::bbNewBasicBlock(BBjumpKinds jumpKind)
 
     return block;
 }
+
+//------------------------------------------------------------------------------
+// DisplayStaticSizes: display various static sizes of the BasicBlock data structure.
+//
+// Arguments:
+//    fout  - where to write the output
+//
+// Return Value:
+//    None
+//
+// Note: This function only does something if MEASURE_BLOCK_SIZE is defined, which it might
+// be in private Release builds.
+//
+/* static */
+void BasicBlock::DisplayStaticSizes(FILE* fout)
+{
+#if MEASURE_BLOCK_SIZE
+
+    BasicBlock* bbDummy = nullptr;
+
+    fprintf(fout, "\n");
+    fprintf(fout, "Offset / size of bbNext                = %3u / %3u\n", offsetof(BasicBlock, bbNext),
```

Comment:
Is it worth keeping all this stuff? There are plenty of structs other than `BasicBlock` for which size/layout info is relevant. But it's not practical to have such code for every struct so why bother with `BasicBlock`? Such information can anyway be extracted from PDB files.

Reply:
It may or may not be. I only updated it because I was "in the code". Normally the stuff is disabled, so nobody sees it until they want to do some optimization work. IMO, it's easier than finding the right PDB-based tool .